src/main/resources/static/js/payment.js

```diff
@@ -63,6 +63,13 @@ function setState() {
         }
     }
 
+    if (inputAuth.value.length > 10)
+    {
+        alert("Authorization number must be 10 or less characters.");
+        btnSave.attr("disabled", true);
+        return;
+    }
+
```

Comment:
This bit of code gets triggered during cash payments and fails with
```TypeError: inputAuth is null```

(Whch is fair, there is no auth number field when you're taking cash)
I'd suggest taking this out, and let `maxlength` on the fields handle the validation. If somebody gets nuts and uses developer tools to bypass that, it'll get caught by the server side validation check in PaymentController.java.

Reply:
Fixed ```payment.js``` & added to ```CHANGES.md```. 

I feel like it'd be cool to have each payment type have their own page. I can mess around with it and try to come up with something after I finish the ```Good First Issue``` tagged issues if you want.

Reply:
That'd be good. Issue #12 is in the same area as well. If you break each page out in to their own file and use different Javascript for each, that issue could be solved for as part of that refactotring.